meta/main.yml

```diff
@@ -1,22 +1,24 @@
 ---
 
 galaxy_info:
-  author: "Wojtek Oledzki"
   description: "Install Kafka server."
   license: MIT
-  min_ansible_version: 2.0
+  min_ansible_version: 2.2
   platforms:
     - name: Ubuntu
       versions:
         - trusty
+        - xenial
+        - bionic
   categories:
     - development
 
 dependencies:
   - role: sansible.users_and_groups
-    users_and_groups:
-      groups:
-        - name: kafka
-      users:
-        - name: kafka
-          group: kafka
+    sansible_users_and_groups_groups:
+      - name: "{{ sansible_kafka_group }}"
+    sansible_users_and_groups_users:
+      - name: "{{ sansible_kafka_user }}"
+        group: "{{ sansible_kafka_group }}"
+
+  - role: sansible.zookeeper
```

Comment:
Zookeeper should only be included for tests, it will live on a separate machine in the real world.

Reply:
That wasn't included because zookeeper had that dependency.